editoast/src/models/paced_train.rs

```diff
@@ -121,7 +137,9 @@ impl PacedTrain {
             speed_limit_tag: self.speed_limit_tag.map(|s| s.into()),
             power_restrictions: self.power_restrictions,
             options: self.options,
-            category: self.main_category.map(|category| category.0),
+            category: self.main_category.map(|category| TrainCategory::Main {
```

Comment:
Shouldn't the `category` field of `TrainSchedul` be renamed `main_category` too? Can be done in a future PR.

Reply:
Yes, i can do it in implementation PR